benchexec/tools/nacpa.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec import result
+from benchexec.tools import template
+
+
+class Tool(template.BaseTool2):
+    """
+    This tool-info module runs nacpa, a parallel portfolio
+    of natively compiled CPAchecker instances.
```

Comment:
*nitpick*: tool-info modules do not actively run anything (they just return information), but because this is a common misconception I would prefer to clear this up here.
```suggestion
    Tool-info module for nacpa, a parallel portfolio
    of natively compiled CPAchecker instances.
```